# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/moz.build
# Commit: 1f3365d25033
# Full Hash: 1f3365d25033f9dbc52b13e8fc4c64f3cdd35946
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 2: Create PNotification via PBackground r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227687
# ==============================================================================

diff -r cc25e5c94813 -r 1f3365d25033 dom/notification/moz.build
--- a/dom/notification/moz.build	Mon Nov 11 20:34:11 2024 +0000
+++ b/dom/notification/moz.build	Mon Nov 11 20:34:11 2024 +0000
@@ -21,6 +21,7 @@
 ]
 
 EXPORTS.mozilla.dom.notification += [
+    "IPCUtils.h",
     "NotificationChild.h",
     "NotificationParent.h",
 ]